### PR TITLE
Hitflash Z fix

### PR DESCRIPTION
**Pull request by fgames9000**
- Minor fix in the hitflash Z position

### CODE REVIEW

File: engine/openbor.c [hunk 1]
```diff
@@ -21671,7 +21671,7 @@ int check_collision(s_collision_check_data* collision_data)
     }
     else
     {
-        collision_data->return_overlap->center_z = 1 + collision_data->target_pos->z;
+        collision_data->return_overlap->center_z = 1 + collision_data->seeker_pos->z; //Kratus (11-2021) Minor fix in the hitflash Z position
```
Comment (DCurrent) [inline engine/openbor.c:21674]: I don't agree with this change. Placing the flash Z position over target when the Z center is more than10 pixels away is intentional behavior and more consistent with backward compatibility. If you want the flash to stay with attacker, we can consider a flag for it.

Comment (fgames9000) [reply engine/openbor.c:21674]: Maybe I didn't add enough details, sorry. This change is to fix the wrong Z position issue, once it is now following the target's position and sometimes it's far away from the real collision point.

I will post some images and more details on the forum.

Comment (DCurrent) [reply engine/openbor.c:21674]: I get what you ate trying to do, and am looking into it myself. We'll get it figured out. It's just this particular change isn't a good fix. It just forces the Z over attacker. That will break a lot of modules that use wide Z boxes.

I'm working on a different formula that hopefully will take care of it.

Comment (fgames9000) [reply engine/openbor.c:21674]: No problem 👍

**Pull request closed without merging**